packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -851,7 +851,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
             log: this.log,
             request,
             session,
-            enqueueLinks: async (enqueueOptions: Partial<EnqueueLinksOptions>) => {
+            enqueueLinks: async (enqueueOptions?: Partial<EnqueueLinksOptions>) => {
```

Comment:
this was actually correct, the options are not optional for BasicCrawler as the `urls` option is required

Reply:
Should be narrowed by this commit, please remove your changes from the PR and rebase to have it here:

https://github.com/apify/crawlee/commit/26a1a9625f4fe3eca327af0e768a218f0deea00c